client/starwhale/cli/board/widgets.py

```diff
@@ -125,12 +125,12 @@ def reload(self) -> None:
         data = self._orderby.sort(self.data)
         for idx, item in enumerate(data):
 
-            def try_render(col: Column) -> t.Any:
-                if col.render:
-                    return col.render(idx, item)
-                return get_field(item, col.key)
+            def try_render(col_: Column, idx_: int, item_: t.Any) -> t.Any:
```

Comment:
Is there no need to modify it here?

Reply:
https://docs.python-guide.org/writing/gotchas/#late-binding-closures